svc/LocalStorageService.js

```diff
@@ -16,6 +16,18 @@ import store from 'store2';
  */
 export class LocalStorageService extends HoistService {
 
+    constructor() {
+        super();
+        errorIf(this.isFake,
+            'Local Storage is not supported in this browser. Transient in-memory storage will ' +
+            'be used for various user state, and lost when the page is closed.'
+        );
+    }
+
+    get isFake() {
```

Comment:
Ok, it took me a while to understand why we're using `isFake`, and not `isUnavailable`:  Because we are falling back to `store`'s transient memory.    A doc comment above this function pointing this out might useful.  We're hoping the penny drops for all developers, if we don't document this.